lib/l10n/app_es.arb

```
@@ -1886,5 +1886,17 @@
   "unlinkTaskTitle": "Desvincular tarea",
   "viewMenuTitle": "Vista",
   "whatsNewDoneButton": "Listo",
-  "whatsNewSkipButton": "Omitir"
-}
\ No newline at end of file
+  "whatsNewSkipButton": "Omitir",
+  "agentEvolutionVersionProposed": "Versión propuesta",
```

RuboCop (1.84.2)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)